src/hints/mod.rs

```diff
@@ -368,6 +412,17 @@ impl HintProcessorLogic for SnosHintProcessor {
     }
 }
 
+pub fn hint_stub(
+    _vm: &mut VirtualMachine,
+    _exec_scopes: &mut ExecutionScopes,
+    _ids_data: &HashMap<String, HintReference>,
+    _ap_tracking: &ApTracking,
+    _constants: &HashMap<String, Felt252>,
+) -> Result<(), HintError> {
+    println!("!!! unimplemented hint !!!");
```

Comment:
This should panic.

Reply:
That prevents the hint tool from running (it attempts to call the hint to see if it is implemented)... 

I altered it to return an error in aaecfd3